src/App.tsx

```diff
@@ -15,33 +14,120 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <div className="App">
-    <button type="button">
-      Start
-    </button>
-
-    <button type="button">
-      Sort alphabetically
-    </button>
-
-    <button type="button">
-      Sort by length
-    </button>
-
-    <button type="button">
-      Reverse
-    </button>
-
-    <button type="button">
-      Reset
-    </button>
-
-    <ul className="Goods">
-      <li className="Goods__item">Dumplings</li>
-      <li className="Goods__item">Carrot</li>
-      <li className="Goods__item">Eggs</li>
-      <li className="Goods__item">...</li>
-    </ul>
-  </div>
-);
+export const App = () => {
+  const goods = goodsFromServer;
+
+  const [isVisible, setVisible] = useState(false);
+  const [isReversed, setIsReversed] = useState(false);
+  const [isSorted, setIsSorted] = useState(false);
+  const [sortBy, setSortBy] = useState('alphabetically');
+
+  const onVisibleGoodsListClick = () => setVisible(!isVisible);
+
+  const onReverseClick = () => setIsReversed(!isReversed);
```

Comment:
onReverseClick - It's a convention for pops naming, in your case, it's  handler function:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/52370890/179032810-bee59e1b-e1d2-4d4a-894a-0f006e2476fd.png">

Reply:
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364

Reply:
Fix this in all places